libraries/botbuilder-ai/README.rst

```diff
@@ -0,0 +1,84 @@
+
+============================
+BotBuilder-AI SDK for Python
+============================
+
+.. image:: https://fuselabs.visualstudio.com/SDK_v4/_apis/build/status/Python/SDK_v4-Python-CI?branchName=master
+   :target:  https://fuselabs.visualstudio.com/SDK_v4/_apis/build/status/Python/SDK_v4-Python-CI
+   :align: right
+   :alt: Azure DevOps status for master branch
+.. image:: https://badge.fury.io/py/botbuilder-ai.svg
+   :target: https://badge.fury.io/py/botbuilder-ai
+   :alt: Latest PyPI package version
+
+Cognitive services extensions for Microsoft BotBuilder.
+
+
+How to Install
+==============
+
+.. code-block:: python
+  
+  pip install botbuilder-ai
+
+
+Documentation/Wiki
+==================
+
+You can find more information on the botbuilder-python project by visiting our `Wiki`_.
+
+Requirements
+============
+
+* `Python >= 3.7.3`_
```

Comment:
We should probably change this to `3.7.0` Azure Linux Web Apps do not yet support `3.7.0`.

As it's only a patch version change nothing should break, so we should be safe...

This should be done across all the READMEs